.env.sample

```diff
@@ -1,54 +1,18 @@
-
```

Comment:
I would remove this file. Default values are already set in their respective files (`env.development`, `.env.production`, ...) and any other variable not used here should be listed in the README after merging this PR.